flag.go

```diff
@@ -828,19 +830,24 @@ func (f *FlagSet) usage() {
 
 func (f *FlagSet) setFlag(flag *Flag, value string, origArg string) error {
 	if err := flag.Value.Set(value); err != nil {
-		return f.failf("invalid argument %q for %s: %v", value, origArg, err)
+		return f.failf("invalid argument %q for %v: %v", value, origArg, err)
+	}
+	if !flag.isSetMultipleTimes && flag.Changed && !strings.Contains(flag.Value.Type(), "Slice") {
+		flag.isSetMultipleTimes = true
+		fmt.Fprintf(os.Stderr, "Flag %q is set multiple times. Using last value\n", flag.Name)
```

Comment:
I think pretty much every flag parser I know of does not consider this an error and none of them warn about it. Do you have an example of a flag parser which complains like this?